src/Ilios/ApiBundle/Resources/swagger/definitions/Academicyear.yml

```diff
@@ -0,0 +1,9 @@
+Academicyear:
```

Comment:
I don't think we can do the entire model as readOnly.  I marked the properties and in the docs it only shows up in GET requests.